config/locales/es.yml

```diff
@@ -110,6 +110,14 @@ es:
       user:
         one: Usuario
         other: '%{count} usuarios'
+  actions:
+    toggle_user: "Enable / Disable User"
+    enable_user: "Enable User"
+    disable_user: "Disable User"
+    confirm_user_deactivation: "Estás seguro que quieres desactivar user con username '%{username}'?"
+    confirm_user_activation: "Estás seguro que quieres activar user con username '%{username}'?"
+    user_disabled_message: "User con username '%{username}' ha sido desactivado."
+    user_enabled_message: "User with username '%{username}' ha sido activado."
```

Comment:
Done, but there were missing more translations, so, I have translated some words from line 114 to line 120 https://github.com/EvyW/leap_web/blob/develop/config/locales/es.yml